plasmapy/diagnostics/thomson.py

```diff
@@ -266,105 +266,105 @@ def spectral_density(
-        An array-like object representing :math:`F_i` (defined above).
-        Must sum to 1.0. Default is [1.0], representing a single
-        ion component.
-
-    ions : `str` or `~plasmapy.particles.particle_class.Particle` or
-           `~plasmapy.particles.particle_collections.ParticleList`,
-           shape (Ni, ), optional
-
-        A list or single instance of `~plasmapy.particles.particle_class.Particle`, or
-        strings convertible to `~plasmapy.particles.particle_class.Particle`,
-        or a `~plasmapy.particles.particle_collections.ParticleList`. All ions
-        must be positively charged. Default is ``'H+'`` corresponding to a
-        single species of hydrogen ions.
-
-    electron_vel : `~astropy.units.Quantity`, shape (Ne, 3), optional
-        Velocity of each electron population in the rest frame. (convertible to m/s)
-        If set, overrides ``electron_vdir`` and ``electron_speed``.
-        Defaults to a stationary plasma [0, 0, 0] m/s.
-
-    ion_vel : `~astropy.units.Quantity`, shape (Ni, 3), optional
-        Velocity vectors for each electron population in the rest frame
-        (convertible to m/s). If set, overrides ``ion_vdir`` and ``ion_speed``.
-        Defaults to zero drift for all specified ion species.
-
-    probe_vec : float `~numpy.ndarray`, shape (3, )
-        Unit vector in the direction of the probe laser. Defaults to
-        [1, 0, 0].
-
-    scatter_vec : float `~numpy.ndarray`, shape (3, )
-        Unit vector pointing from the scattering volume to the detector.
-        Defaults to [0, 1, 0] which, along with the default ``probe_vec``,
+        Total combined number density of all electron populations, in
+        units convertible to m\ :sup:`-3`\ .
+
+    T_e : (Ne,) `~astropy.units.Quantity`, |keyword-only|
+        Temperature of each electron population in units convertible to
+        K or eV, where Ne is the number of electron populations.
+
+    T_i : (Ni,) `~astropy.units.Quantity`, |keyword-only|
+        Temperature of each ion population in units convertible to K or
+        eV, where Ni is the number of ion populations.
+
+    efract : (Ne,) |array_like|, |keyword-only|, optional
+        The ratio of the number density of each electron population to
+        the total electron number density, denoted by :math:`F_e` below.
+        Must sum to one. The default corresponds to a single electron
+        population.
+
+        The ratios of the number density of each ion population
+        The fractional number densities of each electron population,
+        denoted by :math:`F_e` below. Must sum to one. The default
+        corresponds to a single electron population.
+
+    ifract : (Ni,) |array_like|, |keyword-only|, optional
+        The fractional number densities of each ion population, denoted
+        by :math:`F_i` below. Must sum to one. The default corresponds
+        to a single ion population.
+
+    ions : (Ni,) |particle-like|, |keyword-only|, default: "p+"
```

Comment:
Continuing to try out the parameter type descriptions from the [matplotlib dev guide](https://matplotlib.org/stable/devel/documenting_mpl.html#parameter-type-descriptions)...in particular that if there's a simple default that can go in the type line, a parameter does not need to be described as "optional".